### PR TITLE
add HTML for Donation section

### LINKED ISSUE #45: donation with request form

### PR DESCRIPTION
**Pull request by alinamarasca**
*  HTML: for Donation section Close #45
* CSS:  
    * fix navbar font color, size, alignment, logo size adjustment
    *  line height 'our program section'

### CODE REVIEW

**Comment by erkamguresen** [comment]
Hi @alinamarasca . Why did you changed the css file in this issue? Normally if there is an error in a closed issue, you can add an issue and assign yourself it and submit it separately.  Is this one also closes some existing issues?

**Comment by alinamarasca** [comment]
> Hi @alinamarasca . Why did you changed the css file in this issue? Normally if there is an error in a closed issue, you can add an issue and assign yourself it and submit it separately. Is this one also closes some existing issues?

hmmm..should have been two separate PR's.
One - HTML Donation section
Two - few tweaks for CSS. This doesn't solve any issue, but fixes few mistakes in CSS. @erkamguresen  I've edited description so it explains both changes. Do you think an Issue should be created for 'brushing up" CSS file?

File: index.html [hunk 1]
```diff
@@ -157,7 +157,33 @@ <h2>Our Partners</h2>
       <img alt="Open Recognition logo"src="images/2-4-open-recognition.png" />
       <img alt="Bruxelles Formation logo" src="images/2-5-Brussels.png" />
     </div>
-
+<!--Donation-->
+<section class="donation">
+  <div class="options">
+    <div class="message">
+<h2>Support the developers of tomorrow</h2> 
+Help refugees build digital & soft skills for a career in web development. We are a non-profit organization. Thanks to your support we can improve and keep our program free of charge for students.
```
Comment (mayuri2510) [inline index.html:165]: It should be use tag to wrap a paragraph /text.

File: index.html [hunk 2]
```diff
@@ -157,7 +157,33 @@ <h2>Our Partners</h2>
...
+    </div>
+  <div class="donation-form">
```
Comment (mayuri2510) [inline index.html:167]: HTML form element is more suitable than only div for donation form. Form element has lot many benefits and original HYF site also use it.
https://www.bennadel.com/blog/2435-are-html-form-tags-required-to-display-form-input-controls.htm#:~:text=The%20consensus%20seems%20to%20be,without%20including%20a%20FORM%20tag.

**Review by mayuri2510 — COMMENTED** [review]
2 observations.
1. HTML form element is more suitable than only div for donation form. Form element has lot many benefits and original HYF site also use it.
https://www.bennadel.com/blog/2435-are-html-form-tags-required-to-display-form-input-controls.htm#:~:text=The%20consensus%20seems%20to%20be,without%20including%20a%20FORM%20tag.

2:It should use Tag to wrap a paragraph /text.

**Comment by erkamguresen** [comment]
I think an experienced reviewer @otagi  is required for this pr. It seems there are some problems and my review will not make any real progress.

**Comment by otagi** [comment]
I will gladly review this PR. But you should first validate your HTML and CSS with a validator tool like https://validator.w3.org, and fix the reported errors. 

Also, I expect to see perfectly indented code before I start the review 😉. It's easy enough to do by hand, or with an auto-formatter extension in VSCode.

PS: If the validator reports errors on code not in the changes of this PR, don't fix them here. Run the validator on the `main` code, fix the errors on a new branch with a separate PR, review and merge it, then rebase this branch on main.

**Review by sheybusta — APPROVED** [review]

**Comment by otagi** [comment]
Mayuri pointed valid concerns to be fixed. Why was this merged before the concerns could be addressed?

**Comment by sheybusta** [comment]
Sorry @otagi . I merged by mistake.

**Comment by otagi** [comment]
No worries. Mistakes are useful to learn about the workflow best practices. 😉

**Pull request merged**